canonical-websites/build.snapcraft.io
# Updated breadcrumbs component to use vanilla generated styles

## Done

Created new component to replace 'vanilla-inspired' breadcrumbs with vanilla-generated breadcrumbs component.

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Sign in
- Go to builds and builds details pages

Breadcrumbs should look (almost) the same as before and use updated Vanilla styles (directly taken from Vanilla framework).

## Issue / Card

Fixes #869

## Screenshots

<img width="314" alt="screen shot 2017-07-04 at 16 56 56" src="https://user-images.githubusercontent.com/83575/27835331-e220642e-60da-11e7-9eff-70a73d32a079.png">
